### DISCUSSION TITLE
After the update the snippets stopped working

### DISCUSSION THREAD

**Discussion by BolotnikovMS** [zed-industries/zed#13831] [Feature Requests]
Hello, after updating to version 0.142.5, the snippets defined in the file snippets.json stopped working. When calling a snippet, it is not in the body tales.

**Comment by BolotnikovMS** [comment]
oops, sorry, the comma was removed from the settings :)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
